### PR TITLE
Clean up gradle files and dependencies and remove unnecessary code.

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -335,9 +262,4 @@ task codeCoverageReport(type: JacocoReport) {
...
         true
     }
-}
-buildScan {
-    termsOfServiceUrl = 'https://gradle.com/terms-of-service'
-    termsOfServiceAgree = 'yes'
-}
-//buildPlugin.dependsOn check
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (hemikak) [reply build.gradle]: Fixed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/7bf4114363461fb30c977a875efc5571b6d7a929

File: build.gradle [hunk 2]
```diff
@@ -172,13 +158,6 @@ allprojects {
                 password System.getenv("packagePAT")
             }
         }
-//        maven {
```
[the remark this answers is not in the document]
Comment (hemikak) [reply build.gradle:175]: Fixed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/7bf4114363461fb30c977a875efc5571b6d7a929

**Review by hevayo — COMMENTED** [review]
Have we removed @openapi:ClientEndpoint annotation which is no longer relevant

**Comment by hemikak** [comment]
> Have we removed @openapi:ClientEndpoint annotation which is no longer relevant

Removed in https://github.com/ballerina-platform/ballerina-openapi/pull/52/commits/69dd3a0b63e417a45d9af3614d8f920e52fb101e

**Review by hevayo — APPROVED** [review]

**Pull request merged**